src/pymatgen/io/cif.py

```diff
@@ -969,7 +969,7 @@ def _get_structure(
         primitive: bool,
         symmetrized: bool,
         check_occu: bool = False,
-        min_vol: float = 0.5,
+        min_vol: float = 1.0,
```

Comment:
I hope 1 Å³ would be a good default threshold volume to consider a structure unphysical (I prefer a "stricter than it should be" threshold to avoid possible false positive), comment/input welcome